pkg/cmd/pulumi/packagecmd/package_publish.go

```diff
@@ -145,6 +144,18 @@ func (cmd *packagePublishCmd) Run(
 		return fmt.Errorf("failed to get schema: %w", err)
 	}
 
+	// If no readme path is provided, check if there's a readme in the package source or plugin directory we can slurp up.
+	if args.readmePath == "" {
+		readmePath, err := cmd.findReadme(packageSrc)
+		if err != nil {
+			return fmt.Errorf("failed to find readme: %w", err)
+		}
+		args.readmePath = readmePath
+	}
+	if args.readmePath == "" {
+		return errors.New("no readme specified, please provide the path to the readme file or add one to the package")
```

Comment:
Nittiest nit imaginable: I think `README` would be a better casing in general for user messages.

Reply:
I would rephrase it to something like:

```suggestion
		return errors.New("no README found. Please add one named README.md to the package, or use --readme to specify the path")
```